app/com/arpnetworking/rollups/RollupManager.java

```diff
@@ -94,9 +96,18 @@ public Receive createReceive() {
     }
 
     private void executorFinished(final RollupExecutor.FinishRollupMessage message) {
+        final RollupDefinition definition = message.getRollupDefinition();
+        final double latencyNs = (double) Duration.between(definition.getStartTime(), Instant.now()).toNanos();
+
         try (Metrics metrics = _metricsFactory.create()) {
-            metrics.addAnnotation("rollup_metric", message.getRollupDefinition().getDestinationMetricName());
+            metrics.addAnnotation("rollup_metric", definition.getDestinationMetricName());
             metrics.incrementCounter("rollup/manager/executor_finished", 1);
+            metrics.setGauge("rollup/manager/executor_finished/latency_sec", latencyNs / 1e9);
+            metrics.setGauge(
+                    "rollup/manager/executor_finished/latency_periods",
```

Comment:
What is this metric, semantically?

Reply:
(typed up a response, realized that this metric could be improved; updating...)

Reply:
No longer useful!